docs/dev-guide/integration/rest-api/content/orphan.rst

```diff
@@ -183,7 +184,7 @@ Remove and individual orphaned content unit by content type and content id.
 | :response_list:`_`
 
 * :response_code:`202,if the content unit is to be deleted`
-* :response_code:`404,if the content does not exist`
+* :response_code:`404,if the content unit or content type do not exist`
```

Comment:
"does" is still the correct word here, because the subject is a single unit or single type.